tests/console/copy_iso_to_external_drive.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Copy the installation ISO to an external drive
+#    test for bug boo#1040749
+# Maintainer: Sergio Lindo Mansilla <slindomansilla@suse.com>
+
+use base 'btrfs_test';
+use strict;
+use testapi;
+
+sub run() {
+    my ($self) = @_;
+
+    #select_console 'user-console';
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit